modules.d/06rngd/module-setup.sh

```diff
@@ -33,6 +33,8 @@ check() {
 install() {
     inst rngd
     inst_simple "${moddir}/rngd.service" "${systemdsystemunitdir}/rngd.service"
+    # make sure dependant libs are installed too
+    inst_libdir_file opensc-pkcs11.so
```

Comment:
Please make this optional. It's at least not around by default on openSUSE.

Reply:
Found no errors when i run dracut -f without opensc-pkcs11.so on my system. Initramfs regenerated correctly

Reply:
True. This command implies `-o`. @haraldh was this an explicit design choice? Is there a rule to tell which inst rules imply `-o`?

Reply:
Yeah, it was always defined to be sloppy. So, I didn't change the behavior over time.